pkg/gitproviders/github.go

```diff
@@ -61,3 +66,47 @@ func WithGithubProviderHandler(handler GithubProviderHandler, fun func() error)
 	}()
 	return fun()
 }
+
+func (h DefaultGithubProviderHandler) OauthConfig() OauthProviderConfig {
+	return &oauth2.Config{
```

Comment:
I'd prefer this function return error if either of the environment variables are unset instead of returning an OauthProviderConfig that won't work.